gui/default/assets/lang/lang-en.json

```diff
@@ -374,11 +388,15 @@
    "You have no ignored folders.": "You have no ignored folders.",
    "You have unsaved changes. Do you really want to discard them?": "You have unsaved changes. Do you really want to discard them?",
    "You must keep at least one version.": "You must keep at least one version.",
+   "You should never add or change anything locally in a \"{%receiveEncrypted%}\" folder.": "You should never add or change anything locally in a \"{{receiveEncrypted}}\" folder.",
```

Comment:
`locally` is implied, thus delete?

Reply:
I like being explicit, leaves less space for user misinterpretation. And "change location" is a ubiquitous source of misunderstandings (e.g. versioning only applying to remote changes). And this string is used in a modal, no space issues there.